packages/dev-utils/src/blockchain_lifecycle.ts

```diff
@@ -1,21 +1,24 @@
-import { RPC } from './rpc';
+import { Web3Wrapper } from '@0xproject/web3-wrapper';
+import * as Web3 from 'web3';
 
 export class BlockchainLifecycle {
-    private _rpc: RPC;
+    private _web3Wrapper: Web3Wrapper;
     private _snapshotIdsStack: number[];
-    constructor() {
-        this._rpc = new RPC();
+    constructor(web3Orweb3Wrapper: Web3Wrapper | Web3) {
```

Comment:
I don't like this at all. All the instances of `BlockchainLifecycle` that are instantiated with `Web3` are in 0x.js tests, and we also instantiate `web3Wrappers` in those tests. Let's pass in `web3Wrapper` everywhere.